Silicon/NXP/iMX6Pkg/Library/iMX6ClkPwrLib/iMX6ClkPwr.c

```diff
@@ -1736,10 +1550,12 @@ ImxpGetPll2MainClkInfo (
     return EFI_SUCCESS;
   }
 
+  // Per the reference manual, DIV_SELECT controls the PLL Loop Divider.
+  // 0 - Fout=Fref*20
+  // 1 - Fout=Fref*22
   if (PllSysReg.DIV_SELECT == 0) {
     ClockInfo->Frequency = ParentInfo.Frequency * 20;
   } else {
-    ASSERT (PllSysReg.DIV_SELECT == 1);

```

Comment:
Is this assert not valid?

Reply:
Unnecessary since CCM_ANALOG_PLL_SYSn.DIV_SELECT is 1 bit wide